src/Sql/Sql/ChangeLog.md

```diff
@@ -33,7 +33,13 @@
     - `Get-AzSqlServerTransparentDataEncryptionProtector`
     - `Set-AzSqlInstanceTransparentDataEncryptionProtector`
     - `Get-AzSqlInstanceTransparentDataEncryptionProtector`
-
+* Added RestrictOutboundNetworkAccess parameter to following cmdlets
```

Comment:
Please add change log msg under the top of upcoming release section

Reply:
Done!